### PR TITLE
GA & GS SUMO updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes MET sensor additions to GA & GS deploy csvs, as well as Roundabout template revision letters.

- For MET sensor confirmation, refer to: https://docs.google.com/spreadsheets/d/1u74ZBrWE5pZw9_auFhBxFku91GOYXml9geBjmnQ8cLw/edit#gid=559421230 (METBK-CT GLOBAL deploy csvs tab)

- For MET sensor height confirmation, see attached file METBK_Module_Heights_RGT_2023-06-27.xlsx (highlighted Global column)

- For Roundabout template revision letters, see attached file RDB_BuildTemplateRevs.xlsx

[RDB_BuildTemplateRevs.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/12095428/RDB_BuildTemplateRevs.xlsx)
[METBK_Module_Heights_RGT_2023-06-27.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/12095431/METBK_Module_Heights_RGT_2023-06-27.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Look into ingesting METBK-CT recovered instrument data (Redmine #14304, #15493) [DT-180]](https://trello.com/c/sSSfBQmE/180-look-into-ingesting-metbk-ct-recovered-instrument-data-redmine-14304-15493-dt-180)

**Review by sauuyer — COMMENTED** [review]
**GA01SUMO_Deploy.csv**

* METLGR-00034 in deploy csv but not METBK_Module_Heights_RGT_2023-06-27.xlsx or in the Calibratio_CSV_Efforts google sheet
* Seems like FDCHP is missing from this cal csv?

**GS01SUMO_Deploy.csv**
* GS01SUMO-SBD12-08-FDCHPA000 not in Calibration_CSV_Efforts google sheet
* GS01SUMO-SBD12-06-METBKA000 not in Calibration_CSV_Efforts google sheet

**Review by sauuyer — APPROVED** [review]
Approved after slacking with Rebecca

File: deployment/GS01SUMO_Deploy.csv [hunk 1]
```diff
@@ -1,199 +1,256 @@
...
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA002,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METBPR-00271,-54.407168,-89.206037,,-5,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA003,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METHRH-00274,-54.407168,-89.206037,,-5,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA004,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METLWR-00248,-54.407168,-89.206037,,-5,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA005,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METPRC-00243,-54.407168,-89.206037,,-5,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA006,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METSWR-00249,-54.407168,-89.206037,,-5,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA007,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METWND-00243,-54.407168,-89.206037,,-6,4589,,,C
+DY096,,DY112,,GS01SUMO-SBD12-08-FDCHPA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-FDCHPA-74102,-54.407168,-89.206037,,-6,4589,,,C
+
```
Comment (cooleyky) [inline deployment/GS01SUMO_Deploy.csv:256]: looks good

**Review by cooleyky — APPROVED** [review]
All looks good

**Pull request merged**